src/js/index.js

```diff
@@ -200,6 +201,20 @@ export const hpe = deepFreeze({
     gap: 'medium',
     color: 'text-strong',
   },
+  formField: {
+    border: {
+      side: 'all',
+    },
+    content: {
+      pad: {
+        horizontal: 'small',
+        bottom: 'small',
+      },
+      disabled: {
```

Comment:
`disabled` should be outside of `content`.

Reply:
I think you commented on an old commit.
I actually took `disabled` out to base.js, so it shouldn't appear on this theme.
@ericsoderberghp

Reply:
See https://github.com/grommet/grommet/pull/3722/files line 551 on the edited code.